# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1254811807 -7200
#      Tue Oct 06 08:50:07 2009 +0200
# Branch release-3-2-x
# Node ID 8d26637b1e19adbc6d59421da0126e96eece51db
# Parent  b7a669bb15e553fee11073d828c895530ef99d18
Added tag release-3-2-3 for changeset ea2283b9436d

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -41,8 +41,9 @@ a5d1e27ee1f4c3fd66c489e83fdb40f92bf1bf43
 739141cde75a22b9ac9b5b327ea30c61fd357748 ss-3-1-52
 bd1b1fe9c6e952202bf5878cd042a168f6df6ddd ss-3-1-53
 3c3cbe8f18e03c73cc34494c8037d8a45a30bf7d ss-3-1-54
 5e276a0b999747b72af34e72eb63f64640da6399 ss-3-1-55
 a1d20052517a950033896f5a876de8f3ee520138 release-3-2-0
 a1d20052517a950033896f5a876de8f3ee520138 release-3-2-0
 530dc15eb653d538aaab0a139093354fbb6f0c68 release-3-2-0
 6607a20f0ab11a60026b8424df2da361ced53979 release-3-2-2
+ea2283b9436d606f71a2cf1cd3e0853f67741fc7 release-3-2-3
